src/ray/object_manager/plasma/create_request_queue.cc

```diff
@@ -117,6 +117,11 @@ Status CreateRequestQueue::ProcessRequests() {
         // actually freeing up in the object store.
         return Status::ObjectStoreFull("Waiting for grace period.");
       } else {
+        std::string dump = "";
```

Comment:
Why don't we do this in `ReplyToCreateClient`? I think we already have the OOM message there (or delete logs there?)

Reply:
I think it's better to do it here since the reply is asynchronous, so the state may change by the time we reply.

Reply:
Do you think then it makes sense to remove or shorten logs in `ReplyToCreateClient `? It is kind of a duplicated logs (or maybe we can shorten logs in `ReplyToCreateClient `)

Reply:
Maybe then we can shorten logs in `ReplyToCreateClient`? I feel like it is a bit duplicated. Why don't we pass the data size of the object into CreateRequest and log everything here instead? (and make the log in `ReplyToCreateClient` very short or debug)

Reply:
Ah I'm not sure I understand what you mean. I don't think anything is getting logged in `ReplyToCreateClient` right now?

Reply:
Oh I meant this log lines; https://github.com/ray-project/ray/blob/f14f197d42c5dcd7c24bfd68a919c27ed515da29/src/ray/object_manager/plasma/store.cc#L957

Reply:
Ah gotcha, okay I removed that log line and merged it with the new one.